benchmarks/styled-components/README.md

```diff
@@ -0,0 +1,21 @@
+# styled-components Benchmark
```

Comment:
Just "Benchmarks" please, they are not all related to styled-components.

Reply:
Everything was already inside a `styled-components` folder, and the project (Web) only benchmarked `styled-components` against `xstyled`. I figured other folders were going to be created to benchmark other libraries, such as `emotion`. I'll just keep everything inside a single folder.